oriedita-data/src/main/java/oriedita/editor/Colors.java

```diff
@@ -38,6 +38,7 @@ public class Colors {
         add(new Color(255, 0, 0, 75), new Color(255, 0, 0, 75), new Color(255, 0, 0, 75));
         add(new Color(230, 230, 230), new Color(230, 230, 230), new Color(54, 54, 54));
         add(new Color(162, 162, 162), new Color(162, 162, 162), new Color(120, 120, 120)); //placeholder
+        add(Color.gray, new Color(128,128,128,128), new Color(128,128,128,128));
     }
 
```

Comment:
Hello, I'm a beginner developer, I'm interested in the oriedita project, I would like to take part in its development. As a developer, I don’t know much yet, but I would like to understand how the project works and works.

Sorry for posting here, I couldn't find any other way to get in touch.

Reply:
Hi @maks2001916,

There's actually a small and hopefully easy issue in oriedita that I hadn't gotten around to logging yet. 
Happy to support you in the fault-find and patch preparation.

Reply:
Tell me about problem solving.

I want to warn that there may be some misunderstandings, since I do not know English and I translate messages. I hope there will be a minimum.

Reply:
Please see #329

Reply:
Hi @maks2001916,

I'll probably patch that above issue in the next week as it seems you're busy; however there are some other issues in this project we could potentially work on together (e.g SVG import). 

Please reach out to me via the email listed in my GitHub profile.